uwblueprint/supportive-housing
# Resident Date Error

## Notion task link


[Resident Date Error](https://www.notion.so/uwblueprintexecs/Resident-Date-Error-f79143ca494243b98527b23d7c87c1e8)



## Implementation description
* Timezone was being stored with resident move in and move out dates which was causing rendering issues when converting from UTC on the backend to EST on the frontend
* To fix this, I updated the schema for those columns to be of type `Date`, which ignores the timezone
* The timezone was also ignored across the application



## Steps to test
1. Ensure that when you make a resident, the move in date matches what you selected
2. Ensure that when you update the move out date, it matches what you selected 



## What should reviewers focus on?
* 


## Checklist
